### ISSUE TITLE
[Android] Mixpanel not working after upgrading React Native to 0.65.0

### ISSUE THREAD

**Issue by antonio5mins** [mixpanel/mixpanel-react-native#93]
We can't build the project for android.
```
A problem occurred configuring project ':mixpanel-react-native'.
> Could not resolve all artifacts for configuration ':mixpanel-react-native:classpath'.
   > Could not resolve com.android.tools.build:builder-model:3.4.1.
     Required by:
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.build:builder:3.4.1
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.lint:lint-gradle-api:26.4.1
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.build:gradle-api:3.4.1
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.build:builder:3.4.1 > com.android.tools:sdk-common:26.4.1
      > Could not resolve com.android.tools.build:builder-model:3.4.1.
         > Could not get resource 'https://jcenter.bintray.com/com/android/tools/build/builder-model/3.4.1/builder-model-3.4.1.pom'.
            > Could not HEAD 'https://jcenter.bintray.com/com/android/tools/build/builder-model/3.4.1/builder-model-3.4.1.pom'.
               > Read timed out
   > Could not resolve com.android.tools.build:manifest-merger:26.4.1.
     Required by:
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.build:builder:3.4.1
      > Could not resolve com.android.tools.build:manifest-merger:26.4.1.
         > Could not get resource 'https://jcenter.bintray.com/com/android/tools/build/manifest-merger/26.4.1/manifest-merger-26.4.1.pom'.
            > Could not HEAD 'https://jcenter.bintray.com/com/android/tools/build/manifest-merger/26.4.1/manifest-merger-26.4.1.pom'.
               > Read timed out
   > Could not resolve com.android.tools.ddms:ddmlib:26.4.1.
     Required by:
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.build:builder:3.4.1
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.build:builder:3.4.1 > com.android.tools.build:builder-test-api:3.4.1
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.build:builder:3.4.1 > com.android.tools:sdk-common:26.4.1
      > Could not resolve com.android.tools.ddms:ddmlib:26.4.1.
         > Could not get resource 'https://jcenter.bintray.com/com/android/tools/ddms/ddmlib/26.4.1/ddmlib-26.4.1.pom'.
            > Could not HEAD 'https://jcenter.bintray.com/com/android/tools/ddms/ddmlib/26.4.1/ddmlib-26.4.1.pom'.
               > Read timed out
   > Could not resolve com.google.guava:guava:26.0-jre.
     Required by:
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.analytics-library:shared:26.4.1
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.analytics-library:crash:26.4.1
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.lint:lint-gradle-api:26.4.1
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.build:gradle-api:3.4.1
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > androidx.databinding:databinding-compiler-common:3.4.1
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.build:bundletool:0.7.2
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.build:builder:3.4.1 > com.android.tools:common:26.4.1
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.build:builder:3.4.1 > com.android.tools.analytics-library:tracker:26.4.1
      > Skipped due to earlier error
   > Could not resolve com.google.code.gson:gson:2.8.0.
     Required by:
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.analytics-library:shared:26.4.1
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > androidx.databinding:databinding-compiler-common:3.4.1
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.build.jetifier:jetifier-core:1.0.0-beta04
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.build:builder:3.4.1 > com.android.tools:sdklib:26.4.1
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-reflect:1.3.31.
     Required by:
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.lint:lint-gradle-api:26.4.1
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.build:builder:3.4.1 > com.android.tools:sdk-common:26.4.1
      > Skipped due to earlier error
   > Could not resolve com.squareup:javapoet:1.8.0.
     Required by:
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > androidx.databinding:databinding-compiler-common:3.4.1
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-stdlib:1.3.31.
     Required by:
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.31
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.3.31.
     Required by:
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.31
      > Skipped due to earlier error
   > Could not resolve com.google.auto.value:auto-value:1.5.2.
     Required by:
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.build:bundletool:0.7.2
      > Skipped due to earlier error
   > Could not resolve com.google.errorprone:error_prone_annotations:2.2.0.
     Required by:
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.build:bundletool:0.7.2
      > Skipped due to earlier error
   > Could not resolve org.jetbrains.kotlin:kotlin-stdlib:1.3.31.
     Required by:
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.build.jetifier:jetifier-core:1.0.0-beta04
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.build.jetifier:jetifier-processor:1.0.0-beta04
      > Skipped due to earlier error
   > Could not resolve com.google.guava:guava:26.0-jre.
     Required by:
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.google.protobuf:protobuf-java-util:3.4.0
      > Skipped due to earlier error
   > Could not resolve com.google.code.gson:gson:2.8.0.
     Required by:
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.google.protobuf:protobuf-java-util:3.4.0
      > Skipped due to earlier error
   > Could not resolve com.google.guava:guava:26.0-jre.
     Required by:
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.build:builder:3.4.1 > com.android.tools.build:apkzlib:3.4.1
      > Skipped due to earlier error
   > Could not resolve org.glassfish.jaxb:jaxb-runtime:2.2.11.
     Required by:
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.build:builder:3.4.1 > com.android.tools:sdklib:26.4.1 > com.android.tools:repository:26.4.1
      > Skipped due to earlier error
   > Could not resolve com.google.guava:guava:26.0-jre.
     Required by:
         project :mixpanel-react-native > com.android.tools.build:gradle:3.4.1 > com.android.tools.build:builder:3.4.1 > com.android.tools:sdklib:26.4.1 > com.android.tools:repository:26.4.1 > com.google.jimfs:jimfs:1.1
      > Skipped due to earlier error

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.9/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 3m 1s
```

**Comment by iamandiradu** [comment]
Try using this [fork](https://github.com/Jasper-Inc/mixpanel-react-native) until my [PR](https://github.com/mixpanel/mixpanel-react-native/pull/95) is merged.

**Comment by zihejia** [comment]
It's been merged in [v1.3.3 ](https://github.com/mixpanel/mixpanel-react-native/releases/tag/v1.3.3). Closing this one now.
